### ISSUE TITLE
Submit Rancher Desktop Windows to relevant package managers

### ISSUE THREAD

**Issue by sirredbeard** [rancher-sandbox/rd#272]
To make the application more accessible:

* WinGet
* Scoop
* Chocolatey

**Comment by jandubois** [comment]
Do you think this makes sense in the current state already? I would think we would wait until we decide that we are in beta before pushing the app to package managers. Then it would also apply to macOS, e.g. to publish a homebrew cask.

**Comment by mook-as** [comment]
Since the installer requires admin (to install WSL), we may need to install it in non-default repositories.  Alternatively, we may need to build an alternative installer that doesn't require admin access (and doesn't attempt to install WSL).  (The current installer has an embedded manifest to request admin access, so just flags is probably not enough unless we play with `__COMPAT_LAYER`.)

**Comment by bashofmann** [comment]
For homebrew cask: Rancher Desktop was already submitted there a few days ago https://github.com/Homebrew/homebrew-cask/pull/111797.

Unfortunately the cask was called "rancher" instead of "rancher-desktop". I'm currently trying to get it renamed: https://github.com/Homebrew/homebrew-cask/pull/111861

**Comment by jandubois** [comment]
> For homebrew cask: Rancher Desktop was already submitted there a few days ago [Homebrew/homebrew-cask#111797](https://github.com/Homebrew/homebrew-cask/pull/111797).

I also wonder if the `conflicts_with` settings for `helm` and `kubectl-cli` are really necessary. Are we creating these symlinks by default, or are they opt-in?

**Comment by bashofmann** [comment]
Yes, I was wondering the same. I wanted to get the rename PR through first though and not add additional things to it.

**Comment by commitay** [comment]
Even if they're manually created symlinks they still need to be managed by homebrew so they are removed if the cask is uninstalled.

This can be done by listing them in `uninstall delete` or by `binary` stanzas which lets homebrew create them when installed so that they are automatically removed during uninstall.

If an app has an option to create symlinks it's usually preferred that Homebrew be allowed to manage it via `binary` stanzas. This can be disabled via `--no-binaries`.

The conflicts are needed as the CI runners has stuff pre-installed that conflicts with the binaries.

**Comment by sirredbeard** [comment]
> Since the installer requires admin (to install WSL), we may need to install it in non-default repositories.

Chocolatey has a mechanism to handle admin privileges, but @phillipsj knows more about Chocolatey than me.

Scoop is a bit more minimalistic and if a package requires admin level to install it will fail and prompt the user to re-run from elevated PowerShell console AFAIK.

**Comment by mitchellmaler** [comment]
@sirredbeard I looking in chocolatey and didn't see this package. Is that still being added for v1.0?
